docs/how-to-use-proof.md

```diff
@@ -0,0 +1,25 @@
+# A Guide on How to Use Asset Proofs in Scalar DL
+
+This document sets out some guidelines for using Asset Proofs in Scalar DL.
+
+## What is an Asset Proof in Scalar DL ?
+
+An Asset Proof in Scalar DL is a set of information about an asset record that is created along with the asset. It is composed of the following records so that it can be treated as evidence of creation of an asset record by a ledger.
```

Comment:
```suggestion
An Asset Proof in Scalar DL is a set of information about an asset record that is created along with the asset. It is composed of the following records so that it can be treated as evidence of the creation of an asset record by a ledger.
```